### PR TITLE
Addressing Eric V's adn Erik K's telechat reviews

### PR DESCRIPTION
**Pull request by zaheduzzaman**

### CODE REVIEW

File: congestion-control-charter.txt [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-Work on congestion control usually occurs in the TCP Maintenance (TCPM) or
-Transport and Services (TSVWG) working groups. RFC 5033 describes a Best Current
+RFC 5033 describes a Best Current
 Practice to evaluate new congestion control algorithms as Experimental or
-Proposed Standard RFCs. When this workflow was established, TCP was the dominant
-consumer of this work, and proposals typically emerged from research groups that
-had limited ability and experience with running large scale tests to understand
-the implications of their proposals.
+Proposed Standard RFCs. TCP was the dominant
+consumer of this work, and proposals typically discussed in research groups (for example, Internet Congestion Control Research Group - ICCRG).
```
Comment (renghardt) [inline congestion-control-charter.txt]:
```suggestion
consumer of this work, and proposals were typically discussed in research groups (for example, Internet Congestion Control Research Group - ICCRG).

```

File: congestion-control-charter.txt [hunk 2]
```diff
@@ -26,14 +19,13 @@ protocols. The congestion control expertise in the working group would also
 make it a natural venue to take on other work related to indications of
 congestion, such as delay, and potentially queueing algorithms.
 
-Accordingly, CONGRESS is chartered to conduct a review of RFC 5033 and to
+CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should
```
Comment (renghardt) [inline congestion-control-charter.txt]:
```suggestion
CONGRESS is chartered to conduct a revision of RFC 5033. The working group should

```

File: congestion-control-charter.txt [hunk 3]
```diff
@@ -26,14 +19,13 @@ protocols. The congestion control expertise in the working group would also
 make it a natural venue to take on other work related to indications of
 congestion, such as delay, and potentially queueing algorithms.
 
-Accordingly, CONGRESS is chartered to conduct a review of RFC 5033 and to
+CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should
 consider a revision that relaxes requirements to encourage more experiments in
-the IRTF/IETF. Adoption of standards-track work should consider (1) empirical
-evidence of safety and (2) stated intent to deploy by major transport
+the IRTF/IETF. Adoption of standards-track congestion control algorithm should consider (1) empirical
```
Comment (renghardt) [inline congestion-control-charter.txt]:
```suggestion
the IRTF/IETF. Adoption of standards-track congestion control algorithms should consider (1) empirical

```

File: congestion-control-charter.txt [hunk 4]
```diff
@@ -26,14 +19,13 @@ protocols. The congestion control expertise in the working group would also
...
 consider a revision that relaxes requirements to encourage more experiments in
-the IRTF/IETF. Adoption of standards-track work should consider (1) empirical
-evidence of safety and (2) stated intent to deploy by major transport
+the IRTF/IETF. Adoption of standards-track congestion control algorithm should consider (1) empirical
+evidence of safety (for example - avoiding congestion collapse) and (2) stated intent to deploy by major transport
 implementations.
 
-While this charter does not specify further deliverables, CONGRESS is authorized
-to adopt other work relating to Congestion Control and AQM without rechartering.
+CONGRESS is authorized to adopt works relating to Congestion Control and AQM without rechartering.
```
Comment (renghardt) [inline congestion-control-charter.txt]:
```suggestion
CONGRESS is authorized to adopt work relating to Congestion Control and AQM without rechartering.

```

File: congestion-control-charter.txt [hunk 5]
```diff
@@ -85,10 +78,4 @@ congestion control in the Internet, including assessments of compliance with
...
 publishing such documents.
 
-Real-time, media adaptation algorithms for peer to peer communications remain
-the focus of RMCAT, but are in scope after that WG closes.
-
-Once the chartered deliverables are complete, CONGRESS will not remain open
-simply “in case” further work comes along. However, if the working group has
-adopted further work in accordance with the guidelines above, it can recharter,
-add milestones for them, and continue until that work is complete.
+CONGRESS will not remain open simply “in case” further work comes along.
```
Comment (evyncke) [inline congestion-control-charter.txt:85]: It addresses my DISCUSS albeit the sentence looks a little weird like it is.

File: congestion-control-charter.txt [hunk 6]
```diff
@@ -26,14 +19,13 @@ protocols. The congestion control expertise in the working group would also
 make it a natural venue to take on other work related to indications of
 congestion, such as delay, and potentially queueing algorithms.
 
-Accordingly, CONGRESS is chartered to conduct a review of RFC 5033 and to
+CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should
 consider a revision that relaxes requirements to encourage more experiments in
-the IRTF/IETF. Adoption of standards-track work should consider (1) empirical
-evidence of safety and (2) stated intent to deploy by major transport
+the IRTF/IETF. Adoption of standards-track congestion control algorithm should consider (1) empirical
+evidence of safety (for example - avoiding congestion collapse) and (2) stated intent to deploy by major transport
```
Comment (evyncke) [inline congestion-control-charter.txt:28]: Still unsure whether "safety" is the right word, but, at least, the example given is useful.

File: congestion-control-charter.txt (hunk 2 above)
Comment (evyncke) [inline congestion-control-charter.txt]: s/revisionof/revision of/

**Review by renghardt — COMMENTED** [review]
Thanks for collecting these changes! I really appreciate the reduced verbosity.

**Review by ekline — APPROVED** [review]
LGTM, for my part

File: congestion-control-charter.txt (hunk 5 above)
Comment (zaheduzzaman) [reply congestion-control-charter.txt:85]: Thanks.

File: congestion-control-charter.txt (hunk 6 above)
Comment (zaheduzzaman) [reply congestion-control-charter.txt:28]: I am afraid we might not have a better alternative word here :-(

**Comment by zaheduzzaman** [comment]
@martinduke can you merge this PR

**Pull request merged**